config/boot.rb

```diff
@@ -4,3 +4,7 @@
 ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../Gemfile', __FILE__)
 
 require 'bundler/setup' # Set up gems listed in the Gemfile.
+begin
+  require 'bootsnap/setup' if ENV['RAILS_ENV'] != 'production'
```

Comment:
What if we added a local opt-out? Like `BOOTSNAP=false` would also have us skip loading? ex `BOOTSNAP=false bundle exec rails s`

Reply:
Yep, makes sense, added